### DISCUSSION TITLE
press enter can‘t  make multiline shell commands

### DISCUSSION THREAD

**Discussion by allworldg** [Taitava/obsidian-shellcommands#229] [Q&A] [answered]
Version：Obsidian 0.14.6
plugin：0.12.1

like title say , how to solve it .
![image](https://user-images.githubusercontent.com/48782874/170850013-e121b298-3941-4921-9809-8c44c78ab302.png)

**Comment by Taitava** [comment, marked as answer]
Whoops, I haven't been clear in the documentation. Multiline shell commands are currently only available in [0.13.0 beta test](https://github.com/Taitava/obsidian-shellcommands/discussions/228).

Thanks for reporting this, I'll add a remark to this documentation page: https://publish.obsidian.md/shellcommands/Basic+usage/How+to+define+shell+commands

**Reply by allworldg** [reply]
Thanks，expect to 0.13

**Reply by Taitava** [reply]
If you want to try it already now, here are instructions on how you can participate in a beta test: https://github.com/Taitava/obsidian-shellcommands/discussions/98 🙂

**Reply by Taitava** [reply]
Just to notify here too, [`0.13.0` was released a few days ago](https://github.com/Taitava/obsidian-shellcommands/releases/tag/0.13.0).

**Reply by allworldg** [reply]
wow! thanks!
